pkgs/tools/misc/alt-server/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, lib, pkgs, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "AltServer-Linux";
+  version = "0.0.5";
+
+  src = fetchurl {
+    url = "https://github.com/NyaMisty/AltServer-Linux/releases/download/v${version}/AltServer-x86_64";
+    sha256 = "sha256-C+fDrcaewRd6FQMrO443xdDk/vtHycQ5zWLCOLPqF/s=";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
```

Comment:
Missing phase hooks, see https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-phase-hooks.md

Reply:
Could you check that I have implemented this correctly in [this](https://github.com/NixOS/nixpkgs/pull/246161/commits/ac5f658cb2326880f473bcf6959fd043f58fee1e) commit?